src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/IContentBlockViewModel.cs

```diff
@@ -9,10 +10,16 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels
     [JsonDiscriminator(Name = "type")]
     public interface IContentBlockViewModel
```

Comment:
Not sure about that. It's probably more suitable as an interface from a semantics standpooint, although changing it to a record would allow us to avoid re-declaring all the getter/setters. 

Unfortunately, it's also being used everywhere and changing it would necessitate renaming to `ContentBlockViewModel` to fit naming conventions. Don't really want to make all those additional file changes across the entire project really!

Reply:
Nope, understood.